actors/builtin/market/deal_test.go

```diff
@@ -74,6 +79,41 @@ func TestDealLabel(t *testing.T) {
 	assert.Contains(t, err.Error(), "cannot unmarshal into nil pointer")
 
 }
+func TestDealLabelJSON(t *testing.T) {
```

Comment:
Test empty label too for more confidence.

Reply:
i.e. Proposal with Label field unset and defaulting to zero value

Reply:
Yessir